autoload/lsp/symbol.vim

```diff
@@ -291,7 +301,10 @@ export def GotoSymbol(lspserver: dict<any>, location: dict<any>, peekSymbol: boo
     # jump to the file and line containing the symbol
     var wid = fname->bufwinid()
     if wid != -1
-      wid->win_gotoid()
+      # do not jump if cur one is same buf
+      if bufwinid(bufnr()) != wid
```

Comment:
The bufwinid() function returns the window-ID of the first window associated with a buffer.  The 'wid' variable is set to the window ID returned by fname->bufwinid().  So I am not sure what this check is preventing?  If the current window is the correct window, then jumping to it using win_gotoid() should not change anything.  Does jumping to the current window using win_gotoid() changing anything?